fill_lookup.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+
+def fill_lookup(size_bins_edges, impact_bins_edges,
```

Comment:
In principle this function can avoid the use of for loops using `axis=` feature and numpy array broadcasting.  But here I am not really sure how to help I would need to try it